DESCRIPTION

```diff
@@ -24,7 +24,8 @@ Depends:
     ProtGenerics (>= 1.17.2)
 Imports:
     methods,
-    IRanges
+    IRanges,
+    MsCoreUtils
```

Comment:
I am wondering why this is even possible because I add `MsCoreUtils` in https://github.com/rformassspectrometry/Spectra/commit/6518dd8a275f7f8c894f882ce20d4ae3a7ec1419 already.

Reply:
Sorry if I missed it.